articles/history-of-nodejs/history-of-nodejs.md

```diff
@@ -0,0 +1,168 @@
+[source](https://www.simform.com/nodejs-use-case/)
+
+### What is Node.js?
+![Nodejs logo](https://drive.google.com/uc?export=view&id=13udtjL4Xz5YICREzII6hcHiv0xM8a_ds)
+[source](https://www.w3schools.com/nodejs/nodejs_intro.asp)
+
+Node.js is a runtime server environment that uses JavaScript on the server side and [asynchronous programming!](https://www.w3schools.com/nodejs/nodejs_intro.asp). It is a free and open source technology that runs on various platforms (Mac OS X, Unix, Windows, etc.)
+
+- **Node.js can:**
+    - create, open, read, write, delete, and close files on the server
+    - generate the dynamic page content
+    - collect form data
+    - add, delete, change data in your database
+
+- **Features**
+    - Easy to learn
+    - Fast
+    - No Buffering
+    - Consistent source code
+    - Massive Library support
+
+### The Early Beginnings
+Believe it or not, Node.js is only twelve years old. In comparison, JavaScript is 26 years old and the Web is 32 years old.
+[Ryan Dahl wrote Node.js](https://en.wikipedia.org/wiki/Node.js) about thirteen ago, in 2009. At first Node.js supported only Linux and Mac OS X. Dahl led the development and maintenance and later on it was [sponsored by Joyent](https://en.wikipedia.org/wiki/Node.js).
+The most popular web server of the time (Apache) was criticized back in 2009 due to its limited possibilities, because the Apache HTTP Server had to handle various connections concurrently(up to 10,000 or more) which were beyond its capabilities.
+The way of creating code was commonly through sequential programming, when there was either a blocked code in the entire process or an implied multiple execution stacks in cases of simultaneous connections.
+On 8 November 2009 at the inaugural European [JSConf the project was demonstrated by Dahl](https://en.wikipedia.org/wiki/Node.js). Node.js is a combination the V8 JavaScript chrome engine, a low-level I/O API and an event loop.
+
+### The Evolution
+![Nodejs image evolution](https://drive.google.com/uc?export=view&id=1e91n-R0-W4S6UOffTNTnBdqyJKNqzOXN)
+[source](https://nodejs.dev/learn/a-brief-history-of-nodejs)
+
+As many browsers competed to offer users the best performance, JavaScript engines also became considerably better. Because of this major browsers worked hard to find ways to make JavaScript run quicker and ways to offer better support for it.
+It wasn't only luck that made Node.js as popular as it is today, but one could say it was built at the right place and at the right time. It introduces a lot of approaches for JavaScript server-side development and [innovative thinking](https://nodejs.dev/learn/a-brief-history-of-nodejs) that has helped many developers.
+
+Node.js has thousands of open-source libraries and the npm website is where most of them are hosted. There are a lot of developer events and conferences that support the Node.js community, these includes Node Interactive, Node Summit and NodeConf as well as other regional events.
```

Comment:
Id suggest including the npm website to help any developers reading along - that may need it